go/libkb/interfaces.go

```diff
@@ -564,6 +564,14 @@ type UserChangedHandler interface {
 	HandleUserChanged(uid keybase1.UID) error
 }
 
+type ConnectivityMonitorResult int
+
+const (
+	ConnectivityMonitor_YES ConnectivityMonitorResult = iota
+	ConnectivityMonitor_NO
+	ConnectivityMonitor_UNKNOWN
```

Comment:
Actually `go vet` seems ok with it for some reason, I guess I don't know what the rules are.